layouts/partials/tabs.html

```diff
@@ -104,9 +104,81 @@ <h3 class="subtitle is-5 is-muted">{{ $subtitle }}</h3>
-            <p>Data Science Lorem ipsum dolor sit amet, consectetur adipisicing elit. Non a voluptatibus, ex odit totam cumque nihil eos asperiores ea, labore rerum. Doloribus tenetur quae impedit adipisci, laborum dolorum eaque ratione quaerat, eos dicta consequuntur atque ex facere voluptate cupiditate incidunt.</p>
-
-            <p>Data Science Lorem ipsum dolor sit amet, consectetur adipisicing elit. Non a voluptatibus, ex odit totam cumque nihil eos asperiores ea, labore rerum. Doloribus tenetur quae impedit adipisci, laborum dolorum eaque ratione quaerat, eos dicta consequuntur atque ex facere voluptate cupiditate incidunt.</p>
+        <div class="grid-container">
+            <div>
+               <img src="images/content_images/ds-landscape.png" alt="ds-landscape" align="left">
+            </div>
+            <div>
+            <p>
+	      Data Science makes it possible to analyze massive amounts of data
+	      and gain meaningful insights. 
+            </p>
+	    <p>
+              Python has a rich ecosytem of libraries that enable Data Science
+	      workflows. These workflows typically involve the following key
+	      processes:
+                <ul class="content-tab">
+                <li><b>Data Extraction:</b> Pandas, Beautiful Soup</li>
+                <li><b>Data Transformation:</b> Pandas</li>
+                <li><b>Data Manipulation:</b> Seaborn, Matplotlib</li>
+                <li><b>Data Exploration:</b> Scikit-learn, SciPy</li>
+                <li><b>Data Presentation:</b> Matplotlib and <a href="./index.html/#tab-visualisation" target="_blank" rel="noreferrer">several others</a></li>
+              </ul>
+            </p>
+            </div>
+	</div>
+	<p>
+	  All data manipulation begins with gathering or extracting data. <a 
+	  href="https://pandas.pydata.org"> Pandas </a> helps in data discovery
+          and handling, while <a href=
+	  "https://www.crummy.com/software/BeautifulSoup/">Beautiful Soup</a>
+	  helps in gathering and web-scrapping data sets.
+	</p>
+	<p>
+          Before machine learning algorithms can be applied to data, it requires
+	  cleaning and preparation by tools such as Pandas. <a href=
+          "https://seaborn.pydata.org">Seaborn</a> is well known for its <a href=
+	  "https://towardsdatascience.com/how-to-perform-exploratory-data-analysis-with-seaborn-97e3413e841d">
+	  exploratory data analysis (EDA)</a> capabilities, while some of the
+          backbone processes required for machine learning, namely, regression
+	  methods, classification, clustering, model validation and selection are
+	  well served by <a href="https://scikit-learn.org">Scikit-learn</a> and
+	  <a href="https://scikit-learn.org">Scipy</a> (statistical computing).
+	</p>  
+        <p>
+	And of course, <b>NumPy</b> is the foundation of almost all of these tools
+	  used for Data Science.
+        </p>
+	<p>
+	Besides these, <a href="http://keras.io">Keras</a> offers deep learning
```

Comment:
I would suggest to move this paragraph over to the Machine Learning tab - that'd be ML/AI.

Also, Keras is unfortunately end-of-life. It only supports Python 2.7-3.6, and development has stopped. Users should migrate to `tensorflow.keras`, so at this point Keras is no longer worth mentioning unfortunately.

Reply:
Other suggestions for Machine Learning: 
- Visualization: Yellowbrick, Eli5
- ML libraries: XGBoost, LightGBM, CatBoost
- AI libraries: PyTorch, MXNet

Yellowbrick may have a nice visualization to steal, e.g. https://www.scikit-yb.org/en/latest/api/regressor/residuals.html#residuals-plot

Reply:
Done - temporarily parked content in ML page - will edit that next - after DS is merged.

Reply:
See https://github.com/numpy/numpy.org/commit/3a9e5aa731474ec46d5ddc343a7c2822e3b478ad